MediaBrowser.WebDashboard/dashboard-ui/dashboard.html

```diff
@@ -15,12 +15,16 @@ <h3 class="serverNameHeader" style="margin: 0;"></h3>
 
                     </div>
                     <div class="paperList" style="padding: 1em;">
-                        <p id="appVersionNumber">
+                        <p id="productInfo">
+                            <span id="productName">
+                            </span><br />
```

Comment:
Just a style thing, so you can ignore it, but I'd rather the span open and close tags were on the same line and br tag on its own.